lib/perl/Genome/SoftwareResult/User.t

```diff
@@ -68,6 +72,7 @@ my $new_sr = $class->with_registered_users(
             callback => \&_newly_created_callback,
             %users_hash
         );
+UR::Context->commit();
```

Comment:
Do these need to be `UR::Context->process->commit()`?  In which case will these deadlock?